### PR TITLE
More visible up to date contributing page

### PR DESCRIPTION
**Pull request by woutermont**
Following up on the weekly Solid GC [meeting of 2022-12-14](https://github.com/solid/specification/pull/488), I updated the Contributing page, and added it to the main navigation and footer. Since I merged most of the Specification page in it (which was mainly about contributing), I added a script to load the TR table into that page.

### CODE REVIEW

**Comment by timea-solid** [comment]
@woutermont this is a huge PR and quite hard to follow. You added a lot of changes in one PR. I wonder if it is not possible to break it out in more PRs: changes to the navigation, changes to content of different pages. 
P.s. Meanwhile the team page changed and the new changes are not part of this PR.

**Comment by woutermont** [comment]
@timea-solid, it's really just a minor update; the navigation is not changed, just replaced to maintain some balance in the footer/dropdowns.

If you really want to split this up, I can look into it, but I don't know when; I am on sick leave for the foreseable future, and uncertain if/when I will find the energy.

**Comment by timea-solid** [comment]
> @timea-solid, it's really just a minor update; the navigation is not changed, just replaced to maintain some balance in the footer/dropdowns.
> 
> If you really want to split this up, I can look into it, but I don't know when; I am on sick leave for the foreseable future, and uncertain if/when I will find the energy.

This does not have a high prio so please rest. 
It was hard to follow through with the changes and the merges still needed. Also, it is easier to vote on/agree upon as a team if it is broken down. In this case, someone might not agree with the menu moving but agree with your content and vice-versa.

**Comment by woutermont** [comment]
Hi @timea-solid, I am slowely trying to work a bit again. I split this PR up into the following (one per page):

- #795 
- #796 

I left the navigation changes out, even though I still believe them to be necessary in order for people to find the contribution pages (and others) more easily.

I also pushed #794, which fixes some problems with the tooling, which had not been updated in two years.

**Comment by csarven** [comment]
Noting here that this PR (and following specific PRs) is one of the good examples of Solid CG interacting/coordinating with Solid Project / Team following a CG discussion/action.
